### ISSUE TITLE
Matching declarations, definitions, and comments in motor.h/cpp

### ISSUE THREAD

**Issue by gcl8a** [WPIRoboticsEngineering/RBE1001Lib#12]
The templates for declarations and definitions match (e.g., float, int, ...) but the names change from .h to .cpp. cpp allows this, but it also leads to contradictions in the comments, I fear. I would fix them for you, but I don't know how particular you are about names. You mostly have 'miliseconds' as a parameter, but msTimeDuration occasionally (note, I did change one set of comments to match, but that's it).

I'd just go with msTimeDuration, if I were you.

**Comment by madhephaestus** [comment]
yeah, good point. A lot of this code is frankensteined bits of other motor classes i have made over time. It could use a cleaning pass.
